packages/theme-chalk/src/upload.scss

```diff
@@ -124,6 +127,7 @@
   }
 
   @include when(dragover) {
+    padding: 39px 9px;
```

Comment:
Can we set variables for `40px 10px`?  And minus 1px to get 39px instead of write 39px directly.

Reply:
of course